integration-cli/docker_cli_exec_test.go

```diff
@@ -357,6 +357,8 @@ func (s *DockerSuite) TestExecInspectID(c *check.C) {
 
 	// But we should still be able to query the execID
 	sc, body, err := request.SockRequest("GET", "/exec/"+execID+"/json", nil, daemonHost())
+
+	c.Assert(err, checker.IsNil, check.Commentf("failed to query the execID"))
```

Comment:
Since we are checking `sc`, I'm not sure this checking makes sense. Maybe we can remove `err` instead (as in L368)